src/pages/components-gallery/index.js

```diff
@@ -1,13 +1,13 @@
 import { Container, SectionHeader } from '@/ui/containers';
 import { GallerySlider, Testimonials } from '@/ui/sliders';
 
-import { Form } from '@/ui/form';
+import { GeneralForm } from '@/components/ui/forms/general-form';
 import { H4 } from '@/ui/typography';
 import Layout from '@/components/layout';
 import { useRouter } from 'next/router';
 
 /**
- * @description - This page is going to be a gallery of all the components available in the Starter template. The structure is yet to be determined.
+ * @description - Requires a redesign. This page is going to be a gallery of all the components available in the Starter template. The structure is yet to be determined.
```

Comment:
Why? How? This doesn't give anyone enough information to do the redesign. Don't talk to yourself, talk to your team.

Reply:
I removed it. What I initially meant by it was that how we want the UI for the homepage of the Starter template to look like. But, I removed it now.